packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -0,0 +1,174 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:collection/collection.dart';
+import 'package:devtools_app_shared/ui.dart';
+import 'package:flutter/material.dart';
+import 'package:logging/logging.dart';
+
+import '../../../shared/diagnostics/diagnostics_node.dart';
+import '../../../shared/diagnostics_text_styles.dart';
+import '../inspector_controller.dart';
+
+final _log = Logger('properties_view');
+
+class PropertiesView extends StatelessWidget {
+  const PropertiesView({
+    super.key,
+    required this.controller,
+    required this.node,
+  });
+
+  final InspectorController controller;
+  final RemoteDiagnosticsNode node;
+
+  Future<List<RemoteDiagnosticsNode>> loadProperties() async {
```

Comment:
We might want the controller be in charge of this instead of the widget. When the selected node changes, the InspectorController could do this, which may make it easier for us to do things like  cancel the operation in the event that the selection changes before we have fetched the properties. This would also lend itself well to the likely case that this logic may get more complicated once property editing is supported.

Reply:
Good point, this is now controlled by the controller